cmd/cli/mesh_list.go

```diff
@@ -62,11 +63,36 @@ func (l *meshListCmd) run() error {
 	}
 
 	w := newTabWriter(l.out)
-	fmt.Fprintln(w, "MESH NAME\tNAMESPACE\t")
+	nds, _ := l.clientSet.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
+
+	fmt.Fprintf(w, "OSM Deployments on cluster %s\n", nds.Items[0].GetName())
+	fmt.Fprintln(w, "\nMESH NAME\tNAMESPACE\tCONTROLLER PODS\tJOINED NAMESPACES\t")
 	for _, elem := range list.Items {
 		m := elem.ObjectMeta.Labels["meshName"]
 		ns := elem.ObjectMeta.Namespace
-		fmt.Fprintf(w, "%s\t%s\t\n", m, ns)
+		var jNs []string    //joined namespace array
```

Comment:
Should we refactor the whole content as one method? then we only need only one unittest.

Reply:
Are you suggesting to make a single function which returns meshName, namespaces, pods and joined namespaces?

Reply:
Single function which returns `controller's pod` and `joined namespaces`